### ISSUE TITLE
How to extract metadata including geolocation from media files

### ISSUE THREAD

**Issue by evalinenjoki** [section-engineering-education/engineering-education#6623]
*NOTE*: All **1st-time contributors** should know Topic approval and PR submission does NOT guarantee your Topic/Article will be published. Our team of Peer Reviewers and Content Moderators will review all PRs that come in to make sure they adhere to the standard of quality we expect from the EngEd community. 

We expect all community memebers to go through provided examples, resources, and previously published material before submitting content. As a rule of thumb, please only submit articles (pull requests) that are complete, formatted correctly, and include a fully polished article (ready to be published and error free). 

All revisions and edits should be *completed on your own forked repo (as to not take up room in the queue)*. **Any PR that is submitted incomplete (meaning not ready to be published as is) will be closed.** A PR being closed does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.

**NOTE: Any topic suggestion copied/taken/modified from YouTube may Lead to a permanent ban from the community.**

### Topic Suggestion
Enter topic suggestion here...

#### Pre-submission advice
By following all our pre-submission advice and reviewing our Resources folder, you will maximise your chances of your topic being approved.

We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner.

Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published (subject to vary depending on _volume and/or backlog of articles_).

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles that will help you propose and write a successful article.

**Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

### Writing sample(s): 
Include any links or writing samples - to help our team better gauge your writing quality.

#### Proposed title of article
How to extract matadata from videos and JPEG files

#### Proposed article introduction
Photo and video are snapped and recorded by use of different formats. With the photos come along matadata which is in layman's microdata of a whole set of data which makes up the overall data. The article will show users how to extract matadata about time, type of phone and with this we will be able to integrate geo-location data to tracing the specific location where the photo or video was taken from. This will show users how to obtain it by integrating exit explorer,excel and google earth. Upon the user having the image, when he or she follows my laid steps, he or she is able to fully posses all matadata on a photo or video.

#### Key takeaways 
1. How matadata solves ransacking problem on kidnapped individuals.
2. How to interpret matadata on images and videos.
3. The user will learn on aspects of photography and clearing matadata before sharing.
4. The challenges of matadata on laying on bad hands ie people who are unauthorized.

#### Article quality
This article explains generally about matadata on media files and how to interpret matadata to comprehend a scenario or a place or victim. It brings about readability case of understanding, and knowledge understanding about media files.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove*NOTE*: All **1st-time contributors** should know Topic approval and PR submission does NOT guarantee your Topic/Article will be published. Our team of Peer Reviewers and Content Moderators will review all PRs that come in to make sure they adhere to the standard of quality we expect from the EngEd community. 

We expect all community memebers to go through provided examples, resources, and previously published material before submitting content. As a rule of thumb, please only submit articles (pull requests) that are complete, formatted correctly, and include a fully polished article (ready to be published and error free). 

All revisions and edits should be *completed on your own forked repo (as to not take up room in the queue)*. **Any PR that is submitted incomplete (meaning not ready to be published as is) will be closed.** A PR being closed does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.

**NOTE: Any topic suggestion copied/taken/modified from YouTube may Lead to a permanent ban from the community.**

### Topic Suggestion
Enter topic suggestion here...

#### Pre-submission advice
By following all our pre-submission advice and reviewing our Resources folder, you will maximise your chances of your topic being approved.

We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner.

Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published (subject to vary depending on _volume and/or backlog of articles_).

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles that will help you propose and write a successful article.

**Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

### Writing sample(s): 
Include any links or writing samples - to help our team better gauge your writing quality.

#### Proposed title of article
How to extract matadata from videos and JPEG files

#### Proposed article introduction
Photo and video are snapped and recorded by use of different formats. With the photos come along matadata which is in layman's microdata of a whole set of data which makes up the overall data. The article will show users how to extract matadata about time, type of phone and with this we will be able to integrate geo-location data to tracing the specific location where the photo or video was taken from. This will show users how to obtain it by integrating exit explorer,excel and google earth. Upon the user having the image, when he or she follows my laid steps, he or she is able to fully posses all matadata on a photo or video.

#### Key takeaways 
1. How matadata solves ransacking problem on kidnapped individuals.
2. How to interpret matadata on images and videos.
3. The user will learn on aspects of photography and clearing matadata before sharing.
4. The challenges of matadata on laying on bad hands ie people who are unauthorized.

#### Article quality
This article explains generally about matadata on media files and how to interpret matadata to comprehend a scenario or a place or victim. It brings about readability case of understanding, and knowledge understanding about media files.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)
 the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by LinusMuema** [comment]
Hi @evalinenjoki 

You haven't cleared the pre-submission advice and templates. Please format your article suggestion so we can easily make of it. When done, ping the @section-engineering-education/content-moderator-team so we can re-open it and give it a review.

As of now, we will close it to clear the backlog in the issues section.

**Comment by evalinenjoki** [comment]
Please open issue #6623
